calibrateFTsensor.m

```diff
@@ -1,3 +1,9 @@
+clear all
+clear all
+close all
+clc
+
```

Comment:
In previous versions of matlab this would also eliminate any breakpoints you had if you wanted to debug. I do not remember if this behavior is the same now. Do you?

Reply:
Actually, it's not the same behavior, at least in Matlab **R2017b**. So `Clear all ` will not clear the breakpoints. On top of that, I'm clearing all twice because for some reason a single execution wasn't clearing the workspace.